package.json

```diff
@@ -109,6 +109,7 @@
     "react-lazyload": "^2.6.2",
     "react-router-config": "^5.1.1",
     "react-router-dom": "^5.0.1",
+    "semver-compare": "^1.0.0",
```

Comment:
Have the bundle sizes increased with this package being added?

Reply:
Nope its teeny tiny

Reply:
![image](https://user-images.githubusercontent.com/11341355/67386319-a42ab680-f58c-11e9-8cbe-6539d576faef.png)